webapp/channels/Makefile

```diff
@@ -1,17 +1,23 @@
-.PHONY: build dist test run clean stop check-style fix-style run-unit emojis help update-dependencies
+.PHONY: dist emojis help update-dependencies
```

Comment:
I switched to specifying `.PHONY` on individual rules below because I screwed myself up a few times because the Makefile referenced non-existent phony targets which caused it to not fail silently, but I forgot to remove this line. I'll do that in a followup